app/models/report.rb

```diff
@@ -1,5 +1,7 @@
 class Report < ActiveRecord::Base
   LOG_LEVELS = %w[debug info notice warning err alert emerg crit]
+  attr_accessible :host_id, :reported_at, :status, :metrics
```

Comment:
Couple of nitpicks - attr_accessible usually goes beneath includes in our class definitions, and remove the second blank line.